CHANGES.md

```diff
@@ -1,6 +1,12 @@
 Change Log
 ==========
 
+### 1.47 - 2018-07-02
+
+##### Additions :tada:
+* `PostProcessStage` has a `selectedFeatures` property which is an array of primitives used for selectively applying a post-process stage. In the fragment shader, use the function `bool czm_selected(vec2 textureCoordinates` to determine whether or not the stage should be applied at that fragment.
+   * The black-and-white and silhouette stages have per-feature support.
```

Comment:
Provide the specific name of the stage in CesiumJS.

Also, what is the plan for per-feature bloom - isn't that where we want to be?  What is stopping us?

Reply:
I don't think per-feature bloom makes sense. With HDR, you could change the luminance so certain geometry has bloom while other don't.

Reply:
If a user wants to implement mouse-over glow for an object that bleeds into the scene on the edges, how do they do it?  We expose luminance as part of the material and they just change it?  Does it need a blur post-process?